x-pack/plugins/ml/public/application/timeseriesexplorer/get_controls_for_detector.ts

```diff
@@ -14,7 +14,7 @@ import type { JobId, CombinedJob } from '../../../common/types/anomaly_detection
  */
 export function getControlsForDetector(
   selectedDetectorIndex: number,
-  selectedEntities: Record<string, any>,
+  selectedEntities: Record<string, any> | undefined,
```

Comment:
can you please use the `MlEntityField` type instead of `any`

Reply:
Went back to double check and the original `Record<string, any>` type is correct - went back to ensure the types are correct.

Reply:
I don't think `any` is correct. If you don't know the incoming type - it should be `unknown` and covered with a type guard. Otherwise you should be aware of the expected type.